client/my-sites/plugins/plugins-browser-list/index.jsx

```diff
@@ -91,6 +95,12 @@ const PluginsBrowserList = ( {
 							compact={ ! isWide }
 						/>
 					) }
+					{ expandedListLink && (
+						<a className="plugins-browser-list__browse-all" href={ expandedListLink }>
+							{ __( 'Browse All' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Browse all plugins', { context: 'button tooltip'} )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/browse-links)